Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -218,6 +218,9 @@
          <TestPlanReference
             reference = "container:Tests/Smoketest4.xctestplan">
          </TestPlanReference>
+         <TestPlanReference
+            reference = "container:ExperimentIntegrationTests.xctestplan">
```

Comment:
I should have mentioned this before, but our plan in the future is to get rid of this Fennec_Enterprise_XCUITests scheme, so if you can please add this new test plan to the Fennec configuration, similar to the Smoktest plans that we have